src/pages/topics/[slug].tsx

```diff
@@ -25,6 +36,45 @@ const Topic: NextPage<TopicProps> = ({ name, description, ctas }) => {
                 description={description}
                 ctas={ctas}
             />
+            <div
+                style={{
```

Comment:
I think using emotion's `css` prop might be better here versus style? 

https://emotion.sh/docs/object-styles